test/widgets/subscription_list_test.dart

```diff
@@ -316,4 +316,58 @@ void main() {
     checkStreamNameWght(mutedStreamWithUnmutedUnreads.name,     400);
     checkStreamNameWght(mutedStreamWithNoUnmutedUnreads.name,   400);
   });
+
+  group('@-mention marker', () {
+    Iterable<AtMentionMarker> getAtMentionMarkers(WidgetTester tester) {
+      return tester.widgetList<AtMentionMarker>(find.byType(AtMentionMarker));
+    }
+
+    testWidgets('is shown when subscription has unread mentions', (tester) async {
+      final streamWithMentions =   eg.stream();
+      final streamWithNoMentions = eg.stream();
+
+      await setupStreamListPage(tester,
+        subscriptions: [
+          eg.subscription(streamWithMentions),
+          eg.subscription(streamWithNoMentions),
+        ],
+        userTopics: [
+          eg.userTopicItem(streamWithMentions,   'a', UserTopicVisibilityPolicy.none),
+          eg.userTopicItem(streamWithNoMentions, 'b', UserTopicVisibilityPolicy.muted),
```

Comment:
We probably don't want to have this topic muted for this test, so that these channels only differ by the single factor we want to test with.